docs/source/user_guide/from_to_dbms.rst

```diff
@@ -0,0 +1,107 @@
+====================
+From/to other DBMSes
+====================
+.. currentmodule:: databricks.koalas
+
+
+The APIs interacting with other DBMSes in Koalas is slightly different from the ones in pandas
+because Koalas leverages JDBC APIs in PySpark to read and write from/to other DBMSes.
+
+The APIs to read/write from/to external DBMSes are as follows:
+
+.. autosummary::
+
+    read_sql_table
+    read_sql_query
+    read_sql
+
+..
+    TODO: we should implement and document `DataFrame.to_sql`.
+
+Koalas needs a canonical JDBC URL for ``con``, and is able to take extra keyword arguments for `the options in PySpark JDBC APIs <https://spark.apache.org/docs/latest/sql-data-sources-jdbc.html>`_:
+
+.. code-block:: python
+
+    ks.read_sql(..., dbtable="...", driver="", keytab="", ...)
+
+
+Reading and writing DataFrames
+------------------------------
+
+In the example below, you will read and write a table in SQLite.
+
+Firstly, create the ``example`` database as below via Python's SQLite library. This will be read to Koalas later:
+
+.. code-block:: python
+
+    import sqlite3
+
+    con = sqlite3.connect('example.db')
+    cur = con.cursor()
+    # Create table
+    cur.execute(
+        '''CREATE TABLE stocks
+           (date text, trans text, symbol text, qty real, price real)''')
+    # Insert a row of data
+    cur.execute("INSERT INTO stocks VALUES ('2006-01-05','BUY','RHAT',100,35.14)")
+    # Save (commit) the changes
+    con.commit()
+    con.close()
+
+Koalas requires a JDBC driver to read so it requires the driver for your particular database on the Spark's classpath. For SQLite JDBC driver, you can download it, for example, as below:
+
+.. code-block:: bash
+
+    curl -O https://repo1.maven.org/maven2/org/xerial/sqlite-jdbc/3.34.0/sqlite-jdbc-3.34.0.jar
+
+After that, you should add it into your Spark session first. Once you add them, Koalas will automatically detect the Spark session and leverage it.
+
+.. code-block:: python
+
+    import os
+
+    from pyspark.sql import SparkSession
+
+    (SparkSession.builder
+        .master("local")
+        .appName("SQLite JDBC")
+        .config(
+            "spark.jars",
+            "{}/sqlite-jdbc-3.34.0.jar".format(os.getcwd()))
+        .config(
+            "spark.driver.extraClassPath",
```

Comment:
Do we need to set `spark.executor.extraClassPath` as well?

Reply:
Maybe not:

> Extra classpath entries to prepend to the classpath of executors. This exists primarily for backwards-compatibility with older versions of Spark. Users typically should not need to set this option.

https://spark.apache.org/docs/latest/configuration.html#runtime-environment

Reply:
But it might still be needed if the `older versions of Spark` includes `2.4`?

Reply:
I think it's okay to just leave it ... I referred to https://spark.apache.org/docs/3.1.1/sql-data-sources-jdbc.html#content.